### ISSUE TITLE
Azure Storage file name problem

### ISSUE THREAD

**Issue by erdemlal** [jschneier/django-storages#285]
when i tried to save a file on azure it gives a name to the file object with backward slashes, for example **my_dir\other_dir\my_file.pdf**  This causes problems while displaying the file url, i think it should be saved with forward slashes

dajngo default storage returns a file name with forward slashes from the _save method which makes sense

```
        # Store filenames with forward slashes, even on Windows.
        return force_text(name.replace('\\', '/'))
```

**Comment by jschneier** [comment]
Yep it even looks like there is a function in that file `clean_name` that will do it but it is not used. Want to make a PR?

**Comment by erdemlal** [comment]
@jschneier i made a small pull request, #310

**Comment by Bernix01** [comment]
any update on this?

**Comment by bjomi** [comment]
Hi, we had problems with this too. We cloned the repo and applied the fix from the pull request our selves. Would be awesome if the pull request got accepted and could be a part of the next release.

Do you think that would be possible?

**Comment by sww314** [comment]
Closed via #565. If not fixed in this update please reopen.
